diana.navasardyan/repo/Lesson_3/exercise_11/exercise_11.py

```diff
@@ -0,0 +1,7 @@
+#! /usr/bin/env python3
+
+length_of_field = int(input("Enter the length(in feet): "))
+width_of_field = int(input("Enter the width(in feet): "))
+acres_1 = 43560
```

Comment:
CSW: acres_1 is a bad variable name. Use ACRES (uppercase as it is a constant value).